Orange/widgets/data/owrank.py

```diff
@@ -685,6 +685,7 @@ def create_scores_table(self, labels):
         # Reshape to 2d array as Table does not like 1d arrays
         feature_names = feature_names[:, None]
 
+        scores = np.nan_to_num(np.array(scores, dtype=np.float64))
```

Comment:
scores was created a couple of lines higher and is already a np.array. Why not specify dtype at creation?